client/my-sites/marketing/buttons/tray.jsx

```diff
@@ -161,11 +162,21 @@ class SharingButtonsTray extends React.Component {
 	getLimitedButtonsNoticeElement = () => {
 		if ( this.props.limited ) {
 			return (
-				<em className="sharing-buttons-preview__panel-notice">
-					{ this.props.translate( 'Sharing options are limited on private sites.', {
-						context: 'Sharing: Buttons',
-					} ) }
-				</em>
+				<p className="sharing-buttons-preview__panel-notice">
+					<em>
+						{ this.props.translate(
+							'It looks like your site is Private. To get more sharing options make your site Public.',
+							{
+								context: 'Sharing: Buttons',
+							}
+						) }
+					</em>
+					<a href="https://wordpress.com/support/settings/privacy-settings/">
+						{ this.props.translate( 'Check out our guide for more details.', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?